Source/CarthageKit/Project.swift

```diff
@@ -1087,8 +1087,41 @@ private func platformForFramework(_ frameworkURL: URL) -> SignalProducer<Platfor
 				let error = Result<(), NSError>.error(message)
 				return .readFailed(frameworkURL, error)
 			}
+			
+			func sdkNameFromExecutable() -> Any? {
+				guard let executableURL = bundle?.executableURL else {
+					return nil
+				}
+				
+				let otoolTask = Process()
+				otoolTask.launchPath = "/usr/bin/otool"
+				otoolTask.arguments = ["-lv", executableURL.path]
+				
+				let otoolPipe = Pipe()
+				otoolTask.standardOutput = otoolPipe
+				otoolTask.standardError = otoolPipe
+				otoolTask.launch()
+				
+				let grepTask = Process()
+				grepTask.launchPath = "/usr/bin/grep"
```

Comment:
I think it'd be preferable to do this without `grep` since that's the approach we've taken elsewhere.